ansible/playbooks/adhoc/grow_root_vg/filter_plugins/oo_filters.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# vim: expandtab:tabstop=4:shiftwidth=4
+'''
+Custom filters for use in openshift-ansible
+'''
+
+import pdb
+from string import digits
+import json
+
+class Volume(object):
```

Comment:
What is this empty class def used for?

Reply:
leftover cruft, deleted (also angered pylint)